mcs/class/corlib/corlib.dll.sources

```diff
@@ -1594,21 +1594,31 @@ ReferenceSources/AppContextDefaultValues.cs
 ../referencesource/mscorlib/system/threading/waithandle.cs
 ../referencesource/mscorlib/system/threading/waithandleExtensions.cs
 
+../../../external/corefx/src/Common/src/System/Threading/Tasks/RendezvousAwaitable.cs
+../../../external/corert/src/Common/src/System/Collections/Generic/LowLevelDictionary.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/DebuggerSupport.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/DebuggerSupport.Dummy.cs
+../../../external/corefx/src/Common/src/System/Threading/Tasks/BeginEndAwaitableAdapter.cs
+../../../external/corert/src/System.Private.CoreLib/shared/System/Threading/Tasks/TaskCanceledException.cs
```

Comment:
As discussed get the shared bits from `external/corefx/src/Common/src/CoreLib/` and not corert